test/integration/employees_edit_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class EmployeesEditTest < ActionDispatch::IntegrationTest
+
+  def setup
+    @employee = employees(:MyString)
+  end
+ 
+  test "successful edit with friendly forwarding" do
```

Comment:
You don't want to have large test cases like this, where the actual case itself is lengthy and contains several assertions. How will you know what is broken? Test cases fail upon the first assertion failure. Having multiple assertions in a single test means that you can't really figure out what all is broken at once. In addition, it makes you really contain a lot of logic in your test case which will start expanding the test case's purpose, which can also be problematic.

An alternative practice is to keep test cases short and sweet. Specifically, this means that you try to keep the test case to one assertion as much as possible. This will represent a single item/behavior/functionality that you are trying to test. For more assertions you make, simply create another test case. This way, you can detect as much defect/fault/errors as possible at once. In addition, each of your test cases is short and has a very specific purpose. This adds clarity to your test cases and, done well, allows for your test cases to be read like documentation.